src/language/phpLanguage.ts

```diff
@@ -7,7 +7,7 @@ export const phpLanguage: Language = {
   toStepDefinitionExpression(node) {
     // match multiline comment
     const text = node.text
-    const match = text.match(/^(\/\*\*[\s*]*)([\s\S]*)(\n[\s]*\*\/)/)
+    const match = text.match(/@(Given |When |Then )(.*)/)
```

Comment:
The previous regex, whilst complex, matched a * at the start, which I believe is the comment signifier.